test/e2e/scheduling/nvidia-gpus.go

```diff
@@ -225,27 +207,6 @@ var _ = SIGDescribe("[Feature:GPU]", func() {
 var _ = SIGDescribe("[Feature:GPUDevicePlugin]", func() {
```

Comment:
It looks like this whole SIGDescribe block is a duplicate of the one above after this change.

Reply:
No, this is for device plugins, the one above is for testing the older gpu integration.